review/migrations/0002_test_data.py

```diff
@@ -23,8 +21,7 @@ def generate_data(apps, schema_editor):
 
         with transaction.atomic():
             for review, rating, day_care_id, dog_owner_id in test_data:
-                Review(review=review, rating=rating, daycare_id=DayCare.objects.get(pk=day_care_id),
-                       dogowner_id=DogOwner.objects.get(pk=dog_owner_id)).save()
+                Review.create(review=review, rating=rating, daycare_id=day_care_id, dogowner_id=dog_owner_id)
 
     operations = [
         migrations.RunPython(generate_data),
```

Comment:
Shouldn't you update the data migration (as you wrote to me in a comment in another PR) ?
And just run migrate so it will generate a new file?

Reply:
the change is to use the right creation method and not changing the data it self
The reason for this is that the original code didn't run the validations